frontend/src/components/ReadOnlyField.js

```diff
@@ -8,8 +8,8 @@ export default function ReadOnlyField({ label, children }) {
 
   return (
     <>
-      <p className="usa-prose margin-bottom-0 text-bold" data-testid="read-only-label">{label}</p>
-      <p className="usa-prose margin-top-0" data-testid="read-only-value">{children}</p>
```

Comment:
@Andrew565  I think the possible issue here has to do with screen readers. Sorry I missed this on my review.